frontend/src/components/videoControls/components/ObjectDetectionSwitch/index.js

```diff
@@ -17,7 +17,9 @@ const ObjectDetectionControls = observer(({ connected }) => {
 
   const handleNNChange = async () => {
     setNN(!useNN)
-    await restart()
+    if (isPlaying) {
```

Comment:
Would you also like to cover the other ticket #64, which asks to hide this switch in simulation mode? As these 2 are pretty simple fixes.

Reply:
done